moto/glue/models.py

```diff
@@ -288,7 +304,14 @@ def create_schema(
         tags=None,
     ):
         """CrateSchema API"""
-        """Validate Registry Id"""
+        warnings.warn(
```

Comment:
Implementation details (or non-implementations, in this case) sh ould not be exposed as a warning.
This should be part of the docstring instead, and it will show up in the documentation.
```
"""
The following parameters/features are not yet implemented:
....
"""
```